soh/src/overlays/actors/ovl_Bg_Spot01_Idosoko/z_bg_spot01_idosoko.c

```diff
@@ -47,7 +47,8 @@ void BgSpot01Idosoko_Init(Actor* thisx, GlobalContext* globalCtx) {
     Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
     CollisionHeader_GetVirtual(&gKakarikoBOTWStoneCol, &colHeader);
     this->dyna.bgId = DynaPoly_SetBgActor(globalCtx, &globalCtx->colCtx.dyna, &this->dyna.actor, colHeader);
-    if (!LINK_IS_ADULT) {
+    // remove bottom of the well rock for adult in rando if song of storms was played in windmill
```

Comment:
do we want to do this always in rando? or just when doing entrance rando?

Reply:
Same as fire temple rocks, will add the dungeon check here.